scripts/source-maintenance.sh

```diff
@@ -279,6 +282,42 @@ for FILENAME in `git ls-files`; do
 	#
 	grep " DEBUG: section" <${FILENAME} | sed -e 's/ \* DEBUG: //' -e 's%/\* DEBUG: %%' -e 's% \*/%%' | sort -u >>doc/debug-sections.tmp
 
+	#
+	# DEBUG Important message list maintenance
+	#
+
+	# combine debugs() multiline strings
+	gawk 'BEGIN {found=0; dbgLine=""; } {
+		if ($0 ~ /\<debugs\(/)
+		    found = 1;
+		if (found)
+		    dbgLine = dbgLine $0;
```

Comment:
If it is not too difficult, it would be nice to strip trailing `//` comments from each `$0` here, so that the `sed` expressions below do not have to deal with them. This is not important, but the scripts would mishandle cases like the two below (after those messages gain IDs):

```C++
debugs(77, DBG_IMPORTANT, "drained too much"); // should not happen

debugs(93, DBG_CRITICAL, "ERROR: Unknown adaptation service or group name: '" <<
        groupId << "'"); // TODO: fail on failures
```

When considering whether or how to improve this, be mindful of non-comment `//`-containing cases like this one:

```C++
debugs(3, DBG_IMPORTANT, "Initializing https:// proxy context");
```

Perhaps this code can only strip `//[^"]*` comments.